main.go

```diff
@@ -206,10 +206,19 @@ func overrideDefaults(defaults *Options, args []string) {
 func parseDefaultConfigs(parser *flags.Parser) error {
 	app := xdg.App{Name: "yab"}
 
-	configFile := app.ConfigPath("defaults.ini")
-	if _, err := os.Stat(configFile); err != nil {
-		// No defaults file to read
-		return nil
+	// Find the best config path to use, preferring the user's config path and
```

Comment:
Can load multiple files? I imagine if there are multiple files, we'd try the system ones first, then override from the user's configuration?